step-certificates/templates/bootstrap.yaml

```diff
@@ -24,6 +24,11 @@ spec:
       serviceAccountName: {{ include "step-certificates.fullname" . }}-config
       restartPolicy: Never
       volumes:
+      {{- if .Values.ca.bootstrap.rootCA.secret }}
+      - name: {{ include "step-certificates.fullname" . }}-ca-volume
+        secret:
+          secretName: "{{ .Values.ca.bootstrap.rootCA.secret }}"
+      {{- end }}
```

Comment:
So, this will be two volumes, one for the certificate, and one for the key, and the code for one of them would look like:
```yaml
volumes: 
{{ - if .Values.ca.bootstrap.rootCertRef }}
- name: {{ include "step-certificates.fullname" . }}-root-volume
 {{- toYaml .Values.ca.bootstrap.rootCertRef | nindent xx }}
{{- end}}
```
And one similar for the key, but you'll have the flexibility to define both cert and key in just one volume.

Reply:
Based on my reply above it should be just one volume, following standard k8s secret.

Reply:
Just one volume, but being able to mount from different sources. I think this would make it:
```yaml
{{- toYaml .Values.ca.bootstrap.rootRef | nindent xx }}
```

Can you call it `rootRef`? or something similar to make clear that is a reference.